apollo-rx2-support/src/main/java/com/apollographql/apollo3/rx2/toRx2.kt

```diff
@@ -67,14 +68,17 @@ fun ApolloInterceptor.toRx2ApolloInterceptor(scheduler: Scheduler = Schedulers.i
   }
 }
 
+@ApolloExperimental
 fun ApolloStore.toRx2ApolloStore(scheduler: Scheduler = Schedulers.io()) = Rx2ApolloStore(this, scheduler)
 
+@ApolloExperimental
 class Rx2ApolloStore(
     private val delegate: ApolloStore,
     private val scheduler: Scheduler,
 ) {
   private val dispatcher = scheduler.asCoroutineDispatcher()
 
+  @ApolloExperimental
```

Comment:
Do we need to put `@ApolloExperimental` on individual methods if the enclosing class is also `@ApolloExperimental`

Reply:
I was not 100% sure, and after checking, it looks like it's not useful: the method calls are marked with the warning even if they are not annotated - so I've removed them 👍 

<img width="318" alt="Screen Shot 2021-11-08 at 09 40 24" src="https://user-images.githubusercontent.com/372852/140710500-7e8a102f-fca4-4d10-87ac-0c3607aac3ce.png">